force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -340,9 +430,12 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Person createPerson(String name, Integer age) {
         // Create a new instance of the Person class using the provided name and age
-        
+        Person person = new Person(name, age);
+        person.setName(name);
+        person.setAge(age);
```

Comment:
No need to set the name and age again since it's already been done in the constructor:
```suggestion
        Person person = new Person(name, age);
```
You would only do this if you were changing the name and/or age.